src/index.ts

```diff
@@ -16,12 +19,19 @@ const connectDb = db.sequelize.authenticate()
 
 const prepareDb = connectDb
   .then(() => db.migrations(log).up())
+  .then(() => insertSeed())
```

Comment:
This will execute the seed everytime the app starts. Totally fine for testing purposes, but remember to remove it later :)

Reply:
added a query to test whether the seed data is already present in the db